node/p2p/p2p.go

```diff
@@ -13,11 +20,19 @@ type Config struct {
 	// NoAnnounceAddresses - Addresses the P2P subsystem may know about, but that should not be announced/advertised,
 	// as undialable from WAN
 	NoAnnounceAddresses []string
+
+	Network        string
+	Bootstrapper   bool
+	BootstrapPeers []string
+	FriendPeers    []string
```

Comment:
We talked about renaming this to `StablePeers` (also needs docs explaining why they are "stable" relative to other peers).

Reply:
I kinda like FriendPeer still, as softens the technicality of things slightly for UX in config, but regardless of the name, the docs explaining the term is a must. Will do

Reply:
Sure whatever you feel is best. I think as long as we have a distinction between `FriendPeer` vs `AcquaintancePeer` 😂  but I personally like either `StablePeer` or `MutualPeer` or something like that. Whatever it is, there should be a nice distinction between ___Peer and a regular Peer

Reply:
`MutualPeer` describes the thing. I am renaming to this